huaweicloud/services/acceptance/css/data_source_huaweicloud_css_logstash_templates_test.go

```diff
@@ -0,0 +1,93 @@
+package css
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourceCssLogstashTemplates_basic(t *testing.T) {
+	dataSource := "data.huaweicloud_css_logstash_templates.test"
+	rName := acceptance.RandomAccResourceName()
+	dc := acceptance.InitDataSourceCheck(dataSource)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceDataSourceCssLogstashTemplates_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttrSet(dataSource, "system_templates.#"),
+					resource.TestCheckResourceAttrSet(dataSource, "system_templates.0.name"),
+					resource.TestCheckResourceAttrSet(dataSource, "custom_templates.#"),
+					resource.TestCheckResourceAttrSet(dataSource, "custom_templates.0.name"),
+
+					resource.TestCheckOutput("type_filter_is_useful", "true"),
+					resource.TestCheckOutput("sys_tem_name_filter_is_useful", "true"),
+					resource.TestCheckOutput("cus_tem_name_filter_is_useful", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testDataSourceDataSourceCssLogstashTemplates_basic(name string) string {
+	return fmt.Sprintf(`
+%s
+
+data "huaweicloud_css_logstash_templates" "test" {
+  depends_on = [huaweicloud_css_logstash_custom_template.test]
+}
+
+locals {
+  sys_tem_name = data.huaweicloud_css_logstash_templates.test.system_templates[0].name
+  cus_tem_name = data.huaweicloud_css_logstash_templates.test.custom_templates[0].name
```

Comment:
```suggestion
  system_name = data.huaweicloud_css_logstash_templates.test.system_templates[0].name
  custem_name = data.huaweicloud_css_logstash_templates.test.custom_templates[0].name
```
and fix other places